lib/jsdom/living/xmlhttprequest.js

```diff
@@ -0,0 +1,722 @@
+"use strict";
+
+const HTTP_STATUS_CODES = require("http").STATUS_CODES;
+const spawnSync = require("child_process").spawnSync;
+
+const jsdom = require("../../jsdom");
```

Comment:
I don't know how to create an XML document object differently, if you have a better idea... :)